.github/workflows/enzyme-ci.yml

```
@@ -62,4 +62,5 @@ jobs:
       - name: test Enzyme/rustbook
         working-directory: rustbook
         run: |
-          cargo +enzyme test --workspace
+          cargo +enzyme test
+          cargo +enzyme test -p samples-loose-types
```

```suggestion
          ENZYME_LOOSE_TYPES=1 cargo +enzyme test -p samples-loose-types
```
Whoops, this was the whole point. Only relevant once the other PR merges.